app/src/main/java/ai/elimu/vitabu/ui/storybook/ChapterFragment.java

```diff
@@ -194,6 +196,12 @@ public void onClick(@NonNull View widget) {
                                     // Report learning event to the Analytics application (https://github.com/elimu-ai/analytics)
                                     LearningEventUtil.reportWordLearningEvent(word, LearningEventType.WORD_PRESSED, getContext(), BuildConfig.ANALYTICS_APPLICATION_ID);
                                 }
+
+                                @Override
+                                public void updateDrawState(@NonNull TextPaint ds) {
+                                    ds.setColor(Color.BLACK);
+                                    ds.setUnderlineText(true);
```

Comment:
Is it possible to set the color of the underline itself? If so, perhaps we could use something like `colorPrimary` or `colorAccent` in `colors.xml` as the color of the line (to make it a little less visible).

![device-2021-01-09-021915](https://user-images.githubusercontent.com/15718174/104079004-8f861d00-5229-11eb-90f9-5bb615abf230.png)

Reply:
@nya-elimuai I've changed the underline color

![device-2021-01-15-003042](https://user-images.githubusercontent.com/8291318/104661967-04de6b80-56ca-11eb-9198-e7586540b42e.png)